Makefile

```diff
@@ -82,4 +82,4 @@ $(foreach pkg,$(PKGLIST),$(eval $(call build-package,$(pkg))))
 .build-packages: ${PACKAGES}
 
 dev-container:
-	docker run --privileged --rm -it -v "${PWD}:${PWD}" -w "${PWD}" cgr.dev/chainguard/sdk:latest
+	docker run --privileged --rm -it -v "${PWD}:${PWD}" -w "${PWD}" ghcr.io/wolfi-dev/sdk:latest@sha256:515a2c5072753f81ce2cbb81bda54b035aefcdb41d7249070009fc018fecd4c9
```

Comment:
@kaniini This is the reference I was referring to, which I pointed to `wolfi-dev/sdk` instead.  Please LMK if this was in error!

Reply:
```
crane config cgr.dev/chainguard/sdk | jq .created
"2023-01-19T06:08:12Z"
```

Kill it with fire 🔥

Reply:
Nah this is fine